client/src/Movies/SavedList.js

```diff
@@ -1,19 +1,21 @@
 import React, { Component } from 'react';
-
+import { Link } from 'react-router-dom';
 export default class SavedList extends Component {
-  constructor(props) {
-    super(props);
-  }
+	constructor(props) {
+		super(props);
+	}
 
-  render() {
-    return (
-      <div className="saved-list">
-        <h3>Saved Movies:</h3>
-        {this.props.list.map(movie => (
-          <span className="saved-movie">{movie.title}</span>
-        ))}
-        <div className="home-button">Home</div>
-      </div>
-    );
-  }
+	render() {
+		return (
+			<div className="saved-list">
+				<h3>Saved Movies:</h3>
+				{this.props.list.map(movie => (
```

Comment:
Remember that anytime a component is generated from a list of data, we need a `key property`. By using a unique key,  React will be able to keep track of each individual component or, in this case `movie.title`.